examples/2d/frustum_sprites.rs

```diff
@@ -0,0 +1,83 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    prelude::*,
+};
+
+const CAMERA_SPEED: f32 = 1000.0;
+
+pub struct CameraTimer(Timer);
+pub struct Direction(Vec3);
+
+fn main() {
+    App::build()
+        .insert_resource(ReportExecutionOrderAmbiguities)
```

Comment:
To avoid having stuff unrelated to the example, this resource should not be added here